dts/xtensa/intel/intel_s1000.dtsi

```diff
@@ -180,5 +182,16 @@
 
 			status = "disabled";
 		};
+
+		usb: usb@a0000 {
+			compatible = "snps,designware-usb";
```

Comment:
Linux Kernel uses `snps,dwc2` for DWC2 USB controller, `compatible = "snps,dwc2";` would probably better choice.

Reply:
That would go against all other designware compatible names. Sounds like a mistake on linux side, if any.